src/resolve/resolved-dnstls-openssl.c

```diff
@@ -98,11 +110,8 @@ int dnstls_stream_connect_tls(DnsStream *stream, DnsServer *server) {
         if (server->server_name) {
                 r = SSL_set_tlsext_host_name(s, server->server_name);
                 if (r <= 0) {
```

Comment:
no braces around single-line ifs, so all of these and below can now be removed

Reply:
removed braces